src/server/game/Entities/Player/Player.cpp

```diff
@@ -4237,11 +4239,24 @@ void Player::DeleteFromDB(ObjectGuid::LowType lowGuid, uint32 accountId, bool up
             }
         default:
             LOG_ERROR("entities.player", "Player::DeleteFromDB: Unsupported delete method: {}.", charDelete_method);
+            deleted = false;
```

Comment:
I do not think that this flag `deleted` is necessary, because if `deleted = false` the function will `return` so it will never execute the bottom code, right?

Reply:
Good point, I think I read that too fast and mistook the `return;` for a `break;`